captum/attr/_core/feature_permutation.py

```diff
@@ -169,6 +189,58 @@ def attribute(  # type: ignore
                             ablations. These arguments are ignored when using
                             FeatureAblation directly.
                             Default: None
+
+        Returns:
+            *tensor* or tuple of *tensors* of **attributions**:
+            - **attributions** (*tensor* or tuple of *tensors*):
+                        The attributions with respect to each input feature.
+                        If the forward function returns
+                        a scalar value per example, attributions will be
+                        the same size as the provided inputs, with each value
+                        providing the attribution of the corresponding input index.
+                        If the forward function returns a scalar per batch, then
+                        attribution tensor(s) will have first dimension 1 and
+                        the remaining dimensions will match the input.
+                        If a single tensor is provided as inputs, a single tensor is
+                        returned. If a tuple is provided for inputs, a tuple of
+                        corresponding sized tensors is returned.
+
+
+        Examples::
+
+            >>> # SimpleClassifier takes a single input tensor of size Nx4x4,
+            >>> # and returns an Nx3 tensor of class probabilities.
+            >>> net = SimpleClassifier()
+            >>> # Generating random input with size 10 x 4 x 4
+            >>> input = torch.randn(10, 4, 4)
+            >>> # Defining FeaturePermutation interpreter
+            >>> feature_perm = FeaturePermutation(net)
+            >>> # Computes permutation attribution, shuffling each of the 16
+            >>> # scalar input independently.
+            >>> attr = feature_perm.attribute(input, target=1)
+
+            >>> # Alternatively, we may want to ablate features in groups, e.g.
```

Comment:
nit: ` we may want to permute features in groups`

Reply:
Good catch, thanks!